core/dao/group_rollout_relation.go

```diff
@@ -0,0 +1,11 @@
+package dao
+
+import (
+	"context"
+
+	"github.com/teamyapp/cloud/libs/errs"
+)
+
+type GroupRolloutRelation interface {
+	FindRolloutIDsByGroupIDAndSortByOrderedIndex(ct context.Context, groupID uint64) ([]uint64, *errs.Error)
```

Comment:
```suggestion
	FindRolloutIDsByGroupID(ct context.Context, groupID uint64) ([]uint64, *errs.Error)
```
Sorting is considered as part of business logic and should be done in service layer.